### ISSUE TITLE
liveries_scanner: "zipfile.BadZipFile: File is not a zip file"

### ISSUE THREAD

**Issue by spencershepard** [pydcs/dcs#287]
File "dcs\helicopters.py", line 28, in Mi_24V
  File "dcs\liveries_scanner.py", line 167, in init
  File "dcs\liveries_scanner.py", line 197, in initialize
  File "dcs\liveries_scanner.py", line 363, in scan_dcs_installation
  File "dcs\liveries_scanner.py", line 328, in scan_mods_path
  File "dcs\liveries_scanner.py", line 314, in scan_liveries
  File "dcs\liveries_scanner.py", line 269, in scan_zip_file
  File "zipfile.py", line 1267, in init
  File "zipfile.py", line 1334, in _RealGetContents
zipfile.BadZipFile: File is not a zip file
[error.txt](https://github.com/pydcs/dcs/files/10716658/error.txt)

User reports they have a similar error with both RotorOps and Liberation

**Comment by spencershepard** [comment]
I'll get more details from the user on their installed mods and directory structure.

**Comment by spencershepard** [comment]
[tree.txt](https://github.com/pydcs/dcs/files/10716775/tree.txt)

**Comment by Raffson** [comment]
It's because of a mod, that's clear from the stack trace. And it also indicates it's a zipfile that's causing the issue.

**Comment by spencershepard** [comment]
@Raffson agreed it looks like it's because of a mod.  Perhaps we could display the name of bad file or handle the exception.  I've seen two reports from users on my end.  I'll post if we can nail it down to any particular mod.

**Comment by Raffson** [comment]
> @Raffson agreed it looks like it's because of a mod. Perhaps we could display the name of bad file or handle the exception. I've seen two reports from users on my end. I'll post if we can nail it down to any particular mod.

Just took a quick glance at the code and it seems I've misinterpreted/forgot what `scan_mods_path` does, but the comments suggest it can also be in the installation folder. The fact that I can't seem to find a modded livery as a zip file, suggests it's not a mod after all. I'll try reproducing locally after dinner...

**Comment by Raffson** [comment]
@spencershepard Do your users have a modded installation? Trace suggests it's in CoreMods/aircraft where a zip file (or at least a file containing '.zip') is causing the issue.

**Comment by spencershepard** [comment]
@Raffson I asked and this is what I got back for a search for zip files in coremods folder. ![image-1.png](https://user-images.githubusercontent.com/25210766/218330740-9fa0e30c-3710-457a-8b92-1582587dfa52.png)

**Comment by Raffson** [comment]
@spencershepard Could you ask your users to narrow it down to 'CoreMods/aircraft/*/Liveries'?

**Comment by spencershepard** [comment]
@Raffson done.  Also for your interest this is where the discussion is: https://discord.gg/pKcbmxeB
